data/json/items/magazine/40mm.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "id": "belt40mm",
```

Comment:
Standard belt with 32 grenades has 8 encumbrance, and belt with 100 grenades has 25 encumbrance. This looks fine for me.